source/expressions.tex

```diff
@@ -1390,6 +1399,10 @@
 \end{itemize}
 the \grammarterm{id-expression} is transformed into
 a class member access expression using \tcode{(*this)} as the object expression.
+If this transformation occurs
+in the predicate of a precondition assertion of a constructor of \tcode{X}
+or a postcondition assertion of a destructor of \tcode{X},
+the expression is ill-formed.  %JMB: Should we add an xref to dcl.contract.func?
```

Comment:
Why?  Seems sufficiently self-contained.

Reply:
ok, not doing it.